Source/EasyNetQ/QueueConsumerPair.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using EasyNetQ.Consumer;
+using EasyNetQ.Internals;
+using EasyNetQ.Topology;
+
+namespace EasyNetQ
+{
+    public class QueueConsumerPair
+    {
+        #region Factory methods
```

Comment:
I try to avoid regions too, I use it when I want to remind myself that probably that class has too much responsibilities.